src/routes/student.js

```diff
@@ -1,21 +1,47 @@
 import { Router } from "express";
 
-import { getStudents, getStudentsById } from "../dataaccess/student.js";
+import {
+  getStudents,
+  getStudentsById,
+  createStudent,
+} from "../dataaccess/student.js";
+import { body, validationResult } from "express-validator";
 
 const router = Router();
 
-router.get('/', async (req, res) => {
+router.get("/", async (req, res) => {
   const students = await getStudents();
 
   res.json(students);
 });
 
-router.get('/:ci', async (req, res) => {
+router.get("/:ci", async (req, res) => {
   const { id } = req.params;
 
   const studentsById = await getStudentsById({ id });
 
   res.json(studentsById);
 });
 
+router.post(
+  "/",
+  [
+    body("ci").isString().notEmpty(),
+    body("name").isString().notEmpty(),
+    body("lastname").isNumeric().notEmpty(),
+    body("birthdate").isNumeric().notEmpty(),
+  ],
```

Comment:
Para esto hacete un studentSchema tal como authSchema y despues le pasas, tambien agregale el correo y el telefono

```suggestion
validateSchema(studentSchema),
```